doc/programming/samples.rst

```diff
@@ -150,21 +150,53 @@ This code sample simulates the passive dynamics of a given model, renders it off
 values, and saves them into a raw data file that can then be converted into a movie file with tools such as ffmpeg. The
 rendering is simplified compared to :ref:`simulate.cc <saSimulate>` because there is no user interaction, visualization
 options or timing; instead we simply render with the default settings as fast as possible. The dimensions and number of
-multi-samples for the offscreen buffer are specified in the MuJoCo model, while the simulation duration, frames-per-
-second to be rendered (usually much less than the physics simulation rate), and output file name are specified as
-command-line arguments. For example, a 5 second animation at 60 frames per second is created with:
+multi-samples for the offscreen buffer are specified in the MuJoCo model with the visual/global/{offwidth,offheight}
+attributes, while the simulation duration, frames-per-second to be rendered (usually much less than the physics
+simulation rate), and output file name are specified as command-line arguments.
 
 .. code-block:: Shell
 
-     render humanoid.xml 5 60 rgb.out
+   record modelfile duration fps rgbfile [adddepth]
 
-The default humanoid.xml model specifies offscreen rendering with 800x800 resolution. With this information in hand, we
-can compress the (large) raw date file into a playable movie file:
+Where the command line arguments are
+
+.. list-table::
+   :width: 95%
+   :align: left
+   :widths: 1 1 5
+   :header-rows: 1
+
+   * - Argument
+     - Default
+     - Meaning
+   * - ``modelfile``
+     - (required)
+     - path to model
+   * - ``duration``
+     - (required)
+     - duration of the recording in seconds
+   * - ``fps``
+     - (required)
+     - number of frames per second
+   * - ``rgbfile``
+     - (required)
+     - path to raw recording file
+   * - ``adddepth``
+     - 1
+     - add a depth image overlay to the lower left corner (0 for no overlay)
+
+For example, a 5 second animation at 60 frames per second is created with:
 
 .. code-block:: Shell
 
-     ffmpeg -f rawvideo -pixel_format rgb24 -video_size 800x800
-       -framerate 60 -i rgb.out -vf "vflip" video.mp4
+     record humanoid.xml 5 60 rgb.out
+
+The default humanoid.xml model specifies offscreen rendering with 2560x1440 resolution. With this information in hand,
```

Comment:
link to the file?
https://github.com/google-deepmind/mujoco/blob/main/model/humanoid/humanoid.xml